### PR TITLE
Docs for new tenant variables experience

### PR DESCRIPTION
**Pull request by N-lson**
[sc-68255]

Example screenshot

image

### CODE REVIEW

**Review by steve-fenton-octopus — APPROVED** [review]
LGTM! Thanks for turning the page into a redirect, rather than deleting it.

File: src/pages/docs/projects/variables/tenant-variables.mdx [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+description: Tenant variables allow you to specify variables that can have different values for each tenant.
+navOrder: 100
+---
+import TenantsWhichVariableTemplatesApply from 'src/shared-content/tenants/tenants-which-variable-templates-apply.include.md';
+
+Tenant variables allow you to configure [variables](/docs/projects/variables/) that can have different values for each [tenant](/docs/tenants)/environment combination. 
+
+![](/docs/projects/variables/images/tenant-variables.png "width=500")
+
+Tenant variables are provided using variable templates which map a single variable name to different values for each tenant based on what you choose to set. Each variable template can define the data type, name, label, help text, and default value.
```
Comment (felixi-octopus) [inline src/pages/docs/projects/variables/tenant-variables.mdx:15]:
```suggestion
Tenant variables are provided using variable templates which map a single variable name to different values for each tenant. Each variable template can define the data type, name, label, help text, and default value.

What do you think about simplifying this?

File: src/pages/docs/projects/variables/tenant-variables.mdx [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+| **Help text**     | The descriptive help text that will be displayed to provide the user with enough information to accurately provide the value. | A shortened, URL friendly, version of the tenant's name. |
+| **Control type** | You can select one of several different data types. This controls the user interface provided to collect the variable value, and determines how the variable value is interpreted. Note the variable values will be stored and interpreted as text. | Single-line text box, multi-line text box, drop down, checkbox, sensitive/password box, Azure account |
+| **Default value** | The value that will be given to the variable if an actual value is not provided. The default value can contain [variable binding expressions](/docs/projects/variables/variable-substitutions). | `https://#{Tenant.Alias}.myapp.com`      |
+| **Options** | (Only applies when control type is drop down). This defines the list of options available for the user to select from the drop down list. Enter each option on a new line. Use `|` to separate values and display text. | `Value1|Display text 1` <br>`Value2|Display text 2`  |
```
Comment (felixi-octopus) [inline src/pages/docs/projects/variables/tenant-variables.mdx:46]: > Use `|` to separate values

Looks like this needs to be escaped
image

**Review by felixi-octopus — COMMENTED** [review]
Late to the party. Couple of things if you got scope

**Pull request merged**